specification/dimensions/chargesubcategory.md

```diff
@@ -0,0 +1,62 @@
+
+- The ChargeSubcategory MUST be present in the billing data.
+- ChargeSubcategory is of type String and MUST be one of the allowed values.
+- ChargeSubcategory MUST NOT be null or empty when ChargeType is "Usage" and the charge is covered by a commitment.
+  - When a usage charge is covered by a commitment, ChargeSubcategory MUST be "Commitment Used".
+  - When a commitment is not used within the committed period, ChargeSubcategory MUST be "Commitment Unused" for the unused usage charge.
+- ChargeSubcategory MUST NOT be null or empty when ChargeType is "Adjustment".
+  - When ChargeSubcategory is "Refund" or "Credit", the charge MUST be negative.
+  - When an adjustment applies to a specific item, the corresponding FOCUS columns that identify that item MUST NOT be null and MUST match the applicable item details the adjustment pertains to.
+- ChargeSubcategory MUST be null when ChargeType is "Tax" or "Usage" and not covered by a commitment.
+
+## Column ID
+
+ChargeSubcategory
+
+## Display Name
+
+Charge Subcategory
+
+## Description
+
+Charge Subcategory is a detailed descriptor within the billing and usage reports that acts as a supplementary detail to the main ChargeType column.
+
+## Content Constraints
+
+| Constraint      | Value          |
+| :-------------- | :------------- |
+| Column required | True           |
+| Data type       | String         |
+| Allows nulls    | True           |
+| Value format    | list-of-values |
+
+Allowed values for usage:
+
+| Value             | Description                                                                                                                                                                                                                                                                                                                                                |
+| :---------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Commitment Used   | This Charge Subcategory value shows that the usage is covered by and benefiting from discounted rates due to pre-existing commitment agreements, such as Reserved Instances or Savings Plans.                                                                                                                                                              |
+| Commitment Unused | This value would represent the portion of a commitment that has not been used. For example, if an organization has reserved instances but is not utilizing all of them, the unused portion falls under this category. It highlights an area where the organization is not fully leveraging its commitments, which could be a lost cost-saving opportunity. |
```

Comment:
Do we need this line: It highlights an area where the organization is not fully leveraging its commitments, which could be a lost cost-saving opportunity. |